src/sagemaker/amazon/linear_learner.py

```diff
@@ -60,7 +60,7 @@ class LinearLearner(AmazonAlgorithmEstimatorBase):
     unbias_label = hp('unbias_label', isbool, 'A boolean')
     num_point_for_scalar = hp('num_point_for_scalar', (isint, gt(0)), 'An integer greater-than 0')
 
-    def __init__(self, role, train_instance_count, train_instance_type, predictor_type='binary_classifier',
+    def __init__(self, role, train_instance_count, train_instance_type, predictor_type,
```

Comment:
Why move this to a required argument? This will break clients working with the default.

Reply:
This is a required argument and we shouldn't default.

Reply:
Since it is a minor release change we will not break existing calls and I'll leave it then.